src/classes/RecipeRepository.js

```diff
@@ -18,6 +18,7 @@ class RecipeRepository {
     this.filterTerm = '';
     this.tag = '';
     this.currentRecipes = this.recipes;
+    this.filterState = '';
```

Comment:
Added a filterState to hold on to whether we're filtering through all of the recipes or just the user's favorites